.eslintrc.js

```diff
@@ -0,0 +1,45 @@
+module.exports = {
+  extends: ['eslint:recommended', 'google'],
+  env: {
+    serviceworker: true,
+    browser: true,
+    node: true,
+    es6: true,
+  },
+  parserOptions: {
+    ecmaVersion: 2017,
+    sourceType: 'module',
+  },
+  globals: {
+    BroadcastChannel: false,
+    WorkboxSW: false,
+    workbox: false,
+  },
+  rules: {
+    'no-tabs': 2,
```

Comment:
I think this is an oversight in `eslint-config-google`, as Google JS Style definitely requires spaces. I'll update that library (related: https://github.com/google/eslint-config-google/issues/40).